core-bundle/src/DataContainer/Palette.php

```diff
@@ -8,72 +8,243 @@
 
 class Palette implements \Stringable
 {
-    /**
-     * @param array<string, array{fields: array<string>, hide: bool}> $config
-     */
-    private function __construct(private array $config = [])
+    final public const POSITION_BEFORE = 'before';
+
+    final public const POSITION_AFTER = 'after';
+
+    final public const POSITION_PREPEND = 'prepend';
+
+    final public const POSITION_APPEND = 'append';
+
+    private array $config;
+
+    public function __construct(string $palette)
```

Comment:
Should probably also allow an empty palette to create wone from scratch?

Reply:
You could use an empty string then, no?

Reply:
Since I think this is a very uncommon use case, I would prefer to have it always explicitly initialized 🤔

Reply:
I will adjust myself then because that's an obvious choice.